group-actors/src/main/java/org/sunbird/util/ActivityConfigReader.java

```diff
@@ -49,7 +50,7 @@ static void loadActivityConfigMap(InputStream in)
         Class<?> classType = Class.forName((String) config.get("serviceClass"));
         SearchServiceUtil searchServiceUtil = (SearchServiceUtil) classType.newInstance();
         for (String activityType : activityTypeList) {
-          activityServiceConfigMap.put(activityType, searchServiceUtil);
+          activityServiceConfigMap.put(activityType.toLowerCase(), searchServiceUtil);
```

Comment:
Lets correct where this is 'Course' - if the activity filter is doing this, lets lowercase it to 'course'. toLowerCase() in a code-base is not a good sign.

Reply:
Added this because we wanted to implement equalsIgnoreCase rather than specifying a particular case. Currently mobile/portal and KP team has told no change required from our end.

Reply:
if there is no change required and stakeholders have said, let us not add this loweCase here please.

Reply:
Reverted .toLowerCase() change